apps/builder/app/builder/features/style-panel/sections/box-shadows/box-shadow-list.tsx

```diff
@@ -0,0 +1,73 @@
+import type { LayersValue, StyleProperty } from "@webstudio-is/css-data";
+import { Flex } from "@webstudio-is/design-system";
+import type { RenderCategoryProps } from "../../style-sections";
+import { Layer } from "./box-shadow-layer";
+
+type BoxShadowLayerProperies = RenderCategoryProps & {
+  layers: LayersValue;
+  property: StyleProperty;
+};
+
+export const BoxShadowLayersList: React.FC<BoxShadowLayerProperies> = ({
+  property,
+  layers,
+  createBatchUpdate,
+}) => {
+  const handleDeleteLayer = (index: number) => {
+    const batch = createBatchUpdate();
+    const layer = layers.value[index];
+
+    const canLayerBeHidden =
+      layer.type === "tuple" || layer.type === "unparsed";
+    if (!canLayerBeHidden) {
```

Comment:
We try to avoid "!" operator as it's almost not visible in many fonts.

Reply:
he means !before, and instead we use if (canLayerBeHidden === false)